src/config/dynamic-theme-fixes.config

```diff
@@ -8909,6 +8909,23 @@ a[title="Google apps"]
 
 ================================
 
+plumbingforums.com
+
+INVERT
+.node-icon::before
+
+CSS
+div,
+.node-icon {
+    background-color: var(--darkreader-neutral-background) !important;
+}
+.input:focus,
+.input.is-focused::placeholder {
+    background: var(--darkreader-neutral-background) !important;
```

Comment:
Any particular reason why you used `background` property? Otherwise you can merge the selector with the CSS declaration above.

Reply:
Yes, for the input fields, it's a `background-image` that causes the problematic background, so we need to use either `background` or `background-image`, but not `background-color`.  And since we are replacing the image with a color, and not an image, we have to use `background` and not `background-image`.

The alternative is:
```
input:focus,
.input.is-focused::placeholder {
    background-image: none !important;
}
```

and then add those same selectors to the other ruleset.

The code I pushed will have better performance and less memory use than this alternative.

Reply:
Got it 👍🏽